plugins/modules/rabbitmq_user.py

```diff
@@ -103,9 +104,38 @@
     description:
       - C(on_create) will only set the password for newly created users.  C(always) will update passwords if they differ.
     type: str
-    required: false
     default: on_create
     choices: ['on_create', 'always']
+  login_protocol:
+    description:
+      - Specify which TCP/IP protocol will be used.
+    type: str
+    default: http
+    choices: ['http', 'https']
+    aliases: [login_protocol, protocol]
```

Comment:
```suggestion
    aliases: [protocol]
```

Reply:
@arloc thanks for pointing it out! Yes, we should avoid aliases in general and especially at the initial stage.

Reply:
All done and fixed. =]